lib/carto/tree/definition.js

```diff
@@ -209,12 +209,14 @@ tree.Definition.prototype.toXML = function(env, existing) {
 };
 
 tree.Definition.prototype.toJS = function(env) {
+  var DEFAULT_ZOOM_VALUE = 8388607;
```

Comment:
This seems to be defined [here](https://github.com/CartoDB/carto/blob/master/lib/carto/tree/zoom.js#L62).